extension/src/fileSystem/index.ts

```diff
@@ -199,16 +200,30 @@ stages:
   return appendFileSync(dvcYamlPath, pipeline)
 }
 
+export const addPlotToDvcYamlFile = (cwd: string, plotObj: PlotConfigData) => {
+  const dvcYamlFile = `${cwd}/dvc.yaml`
+  const { dataFile, ...plot } = plotObj
+  const plotName = relative(cwd, dataFile)
+  const dvcYamlLines = readFileSync(dvcYamlFile, 'utf8').split('\n')
+  const plotYaml = yaml.stringify({ plots: [{ [plotName]: plot }] }).split('\n')
+
+  // TBD this only works correctly for yaml with 2 space indent and no plots
+  // will adjust for other possibilities in another pr
+  dvcYamlLines.push(...plotYaml)
+  void openFileInEditor(dvcYamlFile)
+  return writeFileSync(dvcYamlFile, dvcYamlLines.join('\n'))
+}
+
 export const getFileExtension = (filePath: string) => parse(filePath).ext
 
 export const relativeWithUri = (dvcRoot: string, uri: Uri) =>
   relative(dvcRoot, uri.fsPath)
 
 export const removeDir = (path: string): void => removeSync(path)
 
-export const loadYaml = <T>(path: string): T | undefined => {
+export const loadYamlAsJs = <T>(path: string): T | undefined => {
```

Comment:
[Q] Don't all of these functions `loadYamlAsJs`, `loadCsv`, `loadTsv` take a file and convert it into a Javascript object? Do we need the `AsJs` suffix?